cardlibrary/ancientstirrings.lua

```diff
@@ -1,14 +1,14 @@
 local stirrings = { -- CARD_ID, NAME, POWER, HEALTH, RARITY,BIO	
 	["Korblox Skeleton"] = {
-		["Id"] = 574198419,
+		["Id"] = 1237896695,
 		["Name"] = "Korblox Skeleton",
 		["Health"] = 300,
 		["Power"] = 100,
 		["Rarity"] = "Uncommon",
 		["AttackEffect"] = "Dash",
 		["Color"] = "Blue", 
 		["Archetype"] = "Skeleton",
-		["Cost"] = {["Blue"] = 1,},
+		["Cost"] = {["Blue"] = 2,},
```

Comment:
The majority has decided to give Serhat can't counterattack (counterblock). Increasing the cost by ONE ICON barely fixes the issue with Serhat/Korblox Skeleton.

Reply:
There's also the possibility of making Korblox Skeleton a token.

Reply:
Making it a token does not solve the problem, it rather halts it. any korblox skeletons still in decks will stay there, but nobody else can put one in their decks